apidoc/Titanium/UI/ListView.yml

```diff
@@ -849,6 +849,9 @@ properties:
 
   - name: separatorInsets
     summary: The insets for list view separators (applies to all cells). This property is applicable on iOS 7 and greater.
+    deprecated:
+        since: "5.2.0"
```

Comment:
There should both be 5.2.0. Ill change tableView as well. As the original PR was merged in 5.2.0